lib/jormungandr/cardano-wallet-jormungandr.cabal

```diff
@@ -187,7 +189,7 @@ test-suite unit
       Cardano.Wallet.Jormungandr.RewardsSpec
       Cardano.Wallet.Jormungandr.TransactionSpec
 
-test-suite jormungandr-integration
+test-suite integration
```

Comment:
Careful here. This change was intended to allow filtering out Jörmungandr integration tests from buildkite.

Reply:
ok, I just find it very clumsy to run tests using `...jormungandr:jormunagndr-integration`. I will revert it then...

Reply:
reverted in https://github.com/input-output-hk/cardano-wallet/pull/1640/commits/e9d860edc854e575712c720405e06aa515ea77f7

Reply:
I've not tried it but you _might_ be able to use to rename back to the slightly shorter component name and use `--skip cardano-wallet-jormungandr:test:integration` in `.buildkite/rebuild.hs`.